components/kyma-environment-broker/internal/process/provisioning/ems_bind.go

```diff
@@ -0,0 +1,157 @@
+package provisioning
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/ptr"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/runtime/components"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/servicemanager"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+type EmsBindStep struct {
+	operationManager *process.ProvisionOperationManager
+}
+
+type eventingOverrides struct {
+	oauthClientId 		string
+	oauthClientSecret   string
+	oauthTokenEndpoint  string
+	publishUrl			string
+	bebNamespace		string
+}
+
+func NewEmsBindStep(os storage.Operations) *EmsBindStep {
+	return &EmsBindStep{
+		operationManager: process.NewProvisionOperationManager(os),
+	}
+}
+
+var _ Step = (*EmsBindStep)(nil)
+
+func (s *EmsBindStep) Name() string {
+	return "EMS_Bind"
+}
+
+func (s *EmsBindStep) Run(operation internal.ProvisioningOperation, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
+	if operation.Ems.Instance.InstanceID == "" {
+		log.Warnf("Ems Provisioning step was not triggered")
+		return operation, 0, nil
```

Comment:
such return statement means everything ok, step finishes. Is it ok?

Reply:
you need to chech the last operation - if the provisioning has beed finished

Reply:
Changed